eraser-map.yaml

```diff
@@ -794,6 +794,16 @@ styles:
         base: points
         texture: pois
         interactive: true
+        
+    icons-overlay:
+        base: points
+        texture: pois
+        interactive: true
+        blend: overlay
```

Comment:
You don't actually need `icons-overlay` because the default blend mode for the points style is overlay. Was the intent to make these icons draw over labels?